bodhi/server/consumers/signed.py

```diff
@@ -23,50 +23,29 @@
 """
 
 import logging
-import pprint
-
-import fedmsg.consumers
 
 from bodhi.server import initialize_db
 from bodhi.server.config import config
 from bodhi.server.models import Build
 from bodhi.server.util import transactional_session_maker
 
+import fedora_messaging
```

Comment:
This is a third party import, so it should be placed above the Bodhi imports for PEP-8.